docs/source/dev/supported_vrplib_fields.rst

```diff
@@ -99,3 +102,9 @@ PyVRP supports the following data sections:
    ``VEHICLES_DEPOT_SECTION``
       Depot assignments for each vehicle, typically used in multi-depot instances.
       Vehicles are assigned to the first depot if this section is not provided.
+
+   ``VEHCILES_FIXED_COST``
+      Fixed cost for each vehicle.
+
+   ``VEHICLES_VARIABLE_COST``
+      Variable cost per unit of distance for each vehicle.
```

Comment:
Maybe something like `VEHICLES_UNIT_DISTANCE_COST`? The literature knows this as _variable cost_, yes, but that is only because it never considered duration-related costs. If we ever want to add that too this'd become ambiguous.